docs/stream.md

```diff
@@ -79,6 +79,28 @@ if __name__ == "__main__":
 stream_engine.remove_stream(stream)
 ```
 
+### Starting the stream with initial offsets
+If you want to start your consumption from certain offsets, you can include that in your stream instantiation.
+
+If you try to seek on a partition or topic that is not assigned to your stream, the code will ignore the seek
+and print out a warning. For example, if you have two consumers that are consuming from different partitions,
+and you try to seek for all of the partitions on each consumer, each consumer will seek for the partitions
+it has been assigned, and it will print out a warning log for the ones it was not assigned.
```

Comment:
This looks much better. The only thing I'm missing is some scenarios. Because this is very specific, it would be nice to know what other "use cases" people have found.